packit_service/worker/jobs.py

```diff
@@ -113,7 +113,10 @@ def process_jobs(self, event: Event) -> Dict[str, HandlerResults]:
                 # check whitelist approval for every job to be able to track down which jobs
                 # failed because of missing whitelist approval
                 whitelist = Whitelist()
-                if not whitelist.check_and_report(event, event.get_project()):
+                project_name = getattr(event, "github_login", None)
+                if project_name and project_name in self.config.admins:
+                    logger.info(f"{project_name} is admin, you shall pass")
+                elif not whitelist.check_and_report(event, event.get_project()):
```

Comment:
Why we don't do the same also in `process_comment_jobs()`?

Reply:
oh right, good point

Reply:
fix't & rebase't